src/index.ts

```diff
@@ -256,7 +256,7 @@ export class Lens<S, A> {
   }
 
   /**
-   * generate a lens from a type and a prop
+   * Returns a lens from a type and a prop
```

Comment:
Minor: since you've changed this, `lens` could be also changed to ``Lens`` in order to be consistent with other comments, but feel free to ignore since it's out of the scope of this PR :slightly_smiling_face: